Pull request

akashcodes23 opened the pull request:
Before submitting this pull request, check the changes to see it's only the changes you made intentionally
If there are changes to other lines you didn't make deliberately, it's possible that your IDE made the changes with a utility like prettier.
Next time, make sure that you only add your changes by using `git add -p` and rather than `git add Contributors.md`

If you're doing something in the checklist below, put an `x` inside `[ ]` so that `- [ ]` becomes `- [x]`

- [x] I had fun going through this tutorial (ノ^o^)ノ and learned on the way ٩(＾◡＾)۶
- [x] There are some things I'd like to improve in this tutorial. I have written them below.
- [x] There were steps where I had errors while following this tutorial. I have written them below.

akashcodes23 committed:
Add Akash G Patil to Contributors list

akashcodes23 committed:
Merge branch 'firstcontributions:main' into main

akashcodes23 committed:
Merge branch 'main' into main

Roshanjossey commented:
Hello, congratulations! You've successfully submitted a pull request. 🎉


If you liked the tutorial, please star this repo by clicking the star button on the top right of this page. <img alt="star screenshot" title="star button" src="https://firstcontributions.github.io/assets/star.png">

# Next steps 
 - Continue contributing: If you're looking for projects to contribute to, checkout our [<img src="https://avatars0.githubusercontent.com/u/65761570?s=88&u=640f39b808c75c6b86460aa907dd030bcca2f3c7&v=4" width="22" title="web app" /> webapp](https://firstcontributions.github.io). 
 - If you want more practice checkout [code contributions](https://github.com/roshanjossey/code-contributions). 
 - Share on social media: You can share this content to help more people.

   - [<img src="https://edent.github.io/SuperTinyIcons/images/svg/bluesky.svg" width="22" title="Bluesky" /> Post on Bluesky](https://bsky.app/intent/compose?text=I%20just%20made%20my%20first%20contribution%20to%20open%20source%20with%20First%20Contributions%21%20%F0%9F%8E%89%20Check%20it%20out%3A%0A%0A%0Ahttps%3A%2F%2Fgithub.com%2Ffirstcontributions%2Ffirst-contributions%2F%0A%0A%0A%23opensource).
   - [<img alt="facebook" title="facebook" src="https://edent.github.io/SuperTinyIcons/images/svg/facebook.svg" width="22"> share](https://www.facebook.com/sharer/sharer.php?u=https://firstcontributions.github.io&quote=Yay%21%20I%20just%20made%20my%20first%20open%20source%20contribution%20with%20First%20Contributions.%20You%20can%20too,%20by%20following%20a%20simple%20tutorial%20at%20https%3A//goo.gl/66Axwe&hashtag=%23OpenSource). 
   - [ <img alt="reddit" title="reddit" src="https://edent.github.io/SuperTinyIcons/images/svg/reddit.svg" width="22"> share](https://www.reddit.com/submit?url=https%3A%2F%2Fgithub.com%2Ffirstcontributions%2Ffirst-contributions&title=Learn%20how%20to%20contribute%20to%20open%20source%20projects%20in%205%20minutes). 
   - [<img alt="linkedin" title="linkedin" src="https://edent.github.io/SuperTinyIcons/images/svg/linkedin.svg" width="22"> post](https://www.linkedin.com/sharing/share-offsite/?url=https://github.com/firstcontributions/first-contributions). 
   - [<img alt="dev.to" title="dev.to" src="https://edent.github.io/SuperTinyIcons/images/svg/dev_to.svg" width="22"> publish](https://dev.to/new?prefill=---%0Atitle%3A%20First%20Contributions%3A%20learn%20how%20to%20contribute%20to%20open%20source%20projects%0Apublished%3A%20true%0Atags%3A%20opensource%2C%20beginners%2C%20tutorial%0A---%0A%0AI%20followed%20the%20hands-on%20tutorial%20in%20the%20Readme%20of%20first%20contributions%20and%20made%20my%20first%20pull%20request%20to%20the%20same%20repo.%0A%0A%0A%7B%25%20embed%20https%3A%2F%2Fgithub.com%2Ffirstcontributions%2Ffirst-contributions%20%25%7D). 
   - [<img src="https://edent.github.io/SuperTinyIcons/images/svg/hackernews.svg" width="18" title="HackerNews" /> post](https://news.ycombinator.com/submitlink?u=https%3A%2F%2Fgithub.com%2Ffirstcontributions%2Ffirst-contributions&t=Show%20HN%3A%20Hands%20on%20tutorial%20for%20open%20source%20contribution).
    -  [ <img alt="twitter" title="twitter" src="https://edent.github.io/SuperTinyIcons/images/svg/twitter.svg" width="22"> tweet](https://twitter.com/intent/tweet?text=Yay%21%20I%20just%20made%20my%20first%20open%20source%20contribution%20with%20@1stcontribution.%20You%20can%20too%20at%20https%3A//goo.gl/66Axwe%0A&hashtags=OpenSource,CodeNewbie). 

We'd love to hear your thoughts about this project. Let us know how we can improve by commenting or opening an issue here.


![celebration gif](https://media1.tenor.com/m/Wxr46U7Cb1EAAAAd/clapping-wow.gif)
![I'm really proud](https://media1.tenor.com/m/Evm9ur4Xng4AAAAd/i%27m-really-proud-monkey-d-luffy.gif)

akashcodes23 commented:
I have updated my branch by fetching the latest changes from upstream/main, resolved the merge conflict, and rebased my commits.
Everything is now up-to-date and ready for merge. 
Thank you!
